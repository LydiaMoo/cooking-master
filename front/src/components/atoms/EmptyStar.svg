<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400">
	<g fill="none" stroke="#F65642" stroke-width="10">
			<path d="M212.683,0l45.238,138.503l142.079,0l-104.723,104.926l56.452,156.571l-139.046,-98.796l-168.026,98.796l68.564,-156.571l-113.221,-104.926l145.054,0l67.629,-138.503Z"/>
	</g>
</svg>