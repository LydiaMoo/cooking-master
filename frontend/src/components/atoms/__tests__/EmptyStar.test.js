import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import EmptyStar from '../EmptyStar.svg';

describe('Star.svg', () => {
  it('should match default snapshot', () => {
    const tree = renderer.create(
      <img src={EmptyStar}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});