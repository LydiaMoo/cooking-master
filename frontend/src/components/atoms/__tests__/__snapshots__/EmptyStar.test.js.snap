// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Star.svg should match default snapshot 1`] = `
<img
  src="EmptyStar.svg"
/>
`;
